glean-core/ffi/src/lib.rs

```diff
@@ -128,10 +141,50 @@ where
     with_glean_mut(|glean| Ok(callback(glean)))
 }
 
+struct FdLogger {
+    pub f: Option<RwLock<File>>,
+}
+
+#[derive(Serialize)]
+struct FdLoggingRecord {
+    level: String,
+    message: String,
+}
+
+static mut FD_LOGGER: FdLogger = FdLogger { f: None };
+
+impl log::Log for FdLogger {
+    fn enabled(&self, _metadata: &log::Metadata) -> bool {
+        true
+    }
+
+    fn log(&self, record: &log::Record) {
+        if self.enabled(record.metadata()) {
```

Comment:
Given that we implemented `enabled` to always return true, we can just skip the check here (and maybe add a note why it's unnecessary?)